full-service/src/json_rpc/v1/api/wallet.rs

```diff
@@ -152,8 +152,8 @@ where
                     (
                         a,
                         Amount {
-                            value: v,
-                            token_id: Mob::ID.to_string(),
+                            value: v.into(),
+                            token_id: Mob::ID.to_string().into(),
                         },
```

Comment:
Optional/Equivalent: Use `Amount::new`
```suggestion
                        Amount::new(v, Mob::ID),
```
(and similarly across this PR)